tests/lib/tools/boot-state

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: bootenv show [var]"
```

Comment:
I think this is missing the `boot-state` command name. It should look something like this IMO:
```
     echo "usage: boot-state <COMMAND>"
```
Then the `bootenv show [var]` should be on the following line.

Reply:
nice catch